src/topology/continuous_function/algebra.lean

```diff
@@ -305,6 +306,34 @@ coe_injective.comm_group _ coe_one coe_mul coe_inv coe_div coe_pow coe_zpow
     exactI λ K hK, uniform_continuous_inv.comp_tendsto_uniformly_on
       (tendsto_iff_forall_compact_tendsto_uniformly_on.mp filter.tendsto_id K hK), } }
 
+-- TODO: rewrite the next three lemmas for products and deduce sum case via `to_additive`, once
+-- definition of `tprod` is in place
+
+/-- If `α` is locally compact, and an infinite sum of functions in `C(α, β)`
+converges to `g` (for the compact-open topology), then the pointwise sum converges to `g x` for
+all `x ∈ α`. -/
+lemma has_sum_apply {α : Type*} {β : Type*} {γ : Type*}
```

Comment:
Is there a section you can place these in the file that already has three types with the right typeclasses?

Reply:
There wasn't, but there probably should have been (there was a long string of lemmas all repeating the declaration ` {α : Type*} {β : Type*} [topological_space α] [topological_space β]` each time). I've streamlined it a bit.